README.md

```diff
@@ -147,6 +147,8 @@ vars:
     using_domain_names:         False         #Disable if you are not using domain names
     using_user_tags:            False         #Disable if you are not using user tags
     using_ticket_form_history:  False         #Disable if you are not using ticket form history
+    using_brands:               False         #Disable if you are not using brands
+    using_organizations:        False         #Disable if you are not using organizations
```

Comment:
Same callout as in source about also adding a note for `organization_tag` here

Reply:
Added.